src/file-cache.ts

```diff
@@ -21,22 +25,22 @@ export class FileCache {
 			this.enabled = false;
 		}
 
-		this.storeCss = cache.create(`vite-plugin-webfont-dl__${version}__css`);
-		this.storeFont = cache.create(`vite-plugin-webfont-dl__${version}__font`);
+		this.storeCss = cacheCreate({ cacheId: this.storeCssId });
+		this.storeFont = cacheCreate({ cacheId: this.storeFontId });
 
 		if (!this.enabled) {
 			this.clear();
 		}
 	}
 
-	get(type: 'css' | 'font', url: string): Buffer | string | undefined {
+	get(type: 'css' | 'font', url: string): FileData | undefined {
 		if (!this.enabled) {
 			return;
 		}
 
 		const cachedFile = type === 'css' ?
-			this.storeCss.getKey(url) :
-			this.storeFont.getKey(url);
+			this.storeCss.getKey<any>(url) :
```

Comment:
Previously this was inferrred as `any`, so I've decided to not improve on that, mainly because trying to find the correct type was somewhat complicated (seems to be some serialized `Buffer` representation?).

If wanted, I could dig into this a little further.